src/Movement/Kinematics/HangprinterKinematics.h

```diff
@@ -50,13 +50,20 @@ class HangprinterKinematics : public RoundBedKinematics
 	OBJECT_MODEL_ARRAY(anchors)
 	OBJECT_MODEL_ARRAY(anchorCoordinates)
 
+	bool IsReachablePyramid(float axesCoords[MaxAxes], AxesBitmap axes) const noexcept;
+	bool IsReachablePrism(float axesCoords[MaxAxes], AxesBitmap axes) const noexcept;
+
 private:
 	// Basic facts about movement system
-	static constexpr size_t HANGPRINTER_AXES = 4;
+	const char* ANCHOR_CHARS = "ABCDEFHIJ"; // Skip the G, since it is reserved in G-code
```

Comment:
It makes sense to me. Only G seemed to conflict when calling M669, which is, I think, the only place where it matters, so ABCDEFHIJ seems to work just fine. Apart from there these labels are only used internally in the C code, but never again in the gcode, I think, but I could be wrong. Or new opcodes could use them in the future or something, so it makes sense to play safe. Your proposal sounds good to me. I will change this, thanks for the suggestion.
It just came to mind that perhaps we could reserve some of the codes in M584 for additional axes too.
UVWABCD.
At some point in the future I would like to add at least 3 additional axes for my 8 anchors machine, since with my setup I should be able to get that "for free" (in terms of hardware), so I would need at least UVW, I think. To play around with non planar printing.
I thought that wasn't relevant since, as said, from my understanding, apart from G669, hangprinter anchor labels will only be used internally in the firmware, not in the gcode.
Which brings me to a question I was meaning to ask @dc42 in the future, or I guess just find the code and try to figure out for myself.
Can G1 accept UVWABCD parameters ? I was assuming yes, looking at other projects that seem to be using more than 3 axes and are using RRF too, like https://github.com/FreddieHong19/Open5x
I have many questions about using more than 3 axes, but I was waiting to research the topic a bit more myself before asking to hopefully ask smarter questions.

If we would skip UVWABCD too, the result would be IJKLOQT...mhmm, did we run out of letters?

Could we call them AN0...AN9 or something else longer than 1 letter per parameter in G669?
Sorry if the latter is a stupid question, I'm still kind of new to G-code.